Troubleshooting-Deployment.md

```diff
@@ -43,3 +43,7 @@ role recycling. Check the logs for more information. Visual Studio provides some
 - Check the logs to see if there is an unhandled exception while initializing.
 - Make sure that the connections strings are correct.
 - Check the Azure Cloud Services troubleshooting pages for more information.
+
+## How to Check Logs
+- Use the cloud explorer in Visual Studio to navigate to the appropriate storage table or blob in the storage account. The WADLogsTable is a good starting point for looking at the logs.
+- You might only be logging errors. If you want informational logs as well, you will need to modify the configuration to set the logging severity level. You can find more information about this in the [Orleans Configuration Guide] (http://dotnet.github.io/orleans/Orleans-Configuration-Guide/).
```

Comment:
I think the link to the config page is useful, nevertheless I would *also* show a small snippet of the programmatic and declarative changes you need to do, since the configuration documentation sub-section is pretty extensive and it might not be clear to the user where to find that information.